roles/horizon/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ horizon:
       - { name: functools32 }
       - { name: python-memcached, version: '1.48' }
       - { name: stevedore, version: '1.5.0' }
-      - { name: "git+https://github.com/davidcusatis/bluebox-horizon-customization#egg=bluebox-horizon-customization"}
+      - { name: "git+https://github.com/blueboxgroup/horizon-customization#egg=horizon-customization"}
```

Comment:
Doesn't the egg still need to be bluebox-horizon-customization, since that's the module we expect to find installed?